src/consts.rs

```diff
@@ -30,13 +30,65 @@ pub const U2FHID_IF_VERSION: u32 = 2; // Current interface implementation versio
 pub const U2FHID_FRAME_TIMEOUT: u32 = 500; // Default frame timeout in ms
 pub const U2FHID_TRANS_TIMEOUT: u32 = 3000; // Default message timeout in ms
 
-// U2FHID native commands
-pub const U2FHID_PING: u8 = TYPE_INIT | 0x01; // Echo data through local processor only
-pub const U2FHID_MSG: u8 = TYPE_INIT | 0x03; // Send U2F message frame
-pub const U2FHID_LOCK: u8 = TYPE_INIT | 0x04; // Send lock channel command
-pub const U2FHID_INIT: u8 = TYPE_INIT | 0x06; // Channel initialization
-pub const U2FHID_WINK: u8 = TYPE_INIT | 0x08; // Send device identification wink
-pub const U2FHID_ERROR: u8 = TYPE_INIT | 0x3f; // Error response
+// CTAPHID native commands
+const CTAPHID_PING: u8 = TYPE_INIT | 0x01; // Echo data through local processor only
+const CTAPHID_MSG: u8 = TYPE_INIT | 0x03; // Send U2F message frame
+const CTAPHID_LOCK: u8 = TYPE_INIT | 0x04; // Send lock channel command
+const CTAPHID_INIT: u8 = TYPE_INIT | 0x06; // Channel initialization
+const CTAPHID_WINK: u8 = TYPE_INIT | 0x08; // Send device identification wink
+const CTAPHID_CBOR: u8 = TYPE_INIT | 0x10; // Encapsulated CBOR encoded message
+const CTAPHID_CANCEL: u8 = TYPE_INIT | 0x11; // Cancel outstanding requests
+const CTAPHID_ERROR: u8 = TYPE_INIT | 0x3f; // Error response
+const CTAPHID_KEEPALIVE: u8 = TYPE_INIT | 0x3b; // Should be sent a an authenticator every 100ms and whenever a status changes
```

Comment:
I think it will be a little more sane in the long run to move x3b above x3f so they're in ascending order.

I don't know what linting we have for Rust, but the ones I know we use for JS and C/C++ will complain or reformat long lines like this comment. Could put the comment on the line above the const, and add "Keepalive" (your choice of capitalization) to the beginning of the comment so it's clear which one it goes with.